src/base-cookie.js

```diff
@@ -23,6 +23,26 @@ const get = (name) => {
   }
 };
 
+const getAll = (name) => {
+  try {
+    const ca = document.cookie.split(';');
+    let values = [];
+    for (let i = 0; i < ca.length; i++) {
+      let c = ca[i];
+      while (c.charAt(0) === ' ') {
+        c = c.substring(1, c.length);
+      }
+      if (c.indexOf(name) === 0) {
+        values.push(c.substring(name.length, c.length));
```

Comment:
this is mostly the same logic as the `get` method, except for pushing each cookie into the values array instead of returning after finding the first one. That behavior is why we had the duplicate cookie problem in the first place